enhancements/ingress/gateway-api-with-cluster-ingress-operator.md

```diff
@@ -334,31 +400,56 @@ spec:
     port: 443
 ```
 
-In this configuration, the Gateway specifies that it uses the
-"istio-ingressgateway" service and defines three listeners with two different
-host names: `*.stage.example.com` and `*.prod.example.com`.  Suppose the
-"istio-ingressgateway" has the following service (again, some extraneous details
-are abbreviated):
+In this configuration, the Gateway defines three listeners with two different
+host names: `*.stage.example.com` and `*.prod.example.com`, as well as two
+different TCP ports: 80 and 443.  For this Gateway, Istio creates the following
+service (again, some extraneous details are abbreviated in the following
+example):
 
 ```yaml
 apiVersion: v1
 kind: Service
 metadata:
-  name: istio-ingressgateway
+  labels:
+    gateway.istio.io/managed: istio.io-gateway-controller
+  name: example-gateway
   namespace: openshift-ingress
+  ownerReferences:
+  - apiVersion: gateway.networking.k8s.io/v1alpha2
+    kind: Gateway
+    name: gateway
 spec:
   ports:
-  - name: http
+  - name: status-port
+    port: 15021
+    protocol: TCP
+  - name: stage-http
     port: 80
-  - name: https
+    protocol: TCP
+  - name: stage-https
     port: 443
+    protocol: TCP
+  selector:
+    istio.io/gateway-name: gateway
   type: LoadBalancer
 status:
   loadBalancer:
     ingress:
     - hostname: lb.example.com
 ```
 
+Note that the service defines each port once.  When multiple listeners specify
```

Comment:
I'm good with this wording/content regarding loadbalancer/services, but one of my nit picks is that it's under the `#### New Controller to Manage DNS Records for Gateway Listeners` section, which seems different. Is it worth creating a `Load Balancer Management` section? Though, I won't push back if you want to re-organize this up later.

Reply:
This section describes the implementation details, and I've kept it focused on the changes required in cluster-ingress-operator.  Earlier I was thinking there would be a controller in cluster-ingress-operator for managing the service, but since Istio handles that, I removed that section and just put this example here to show what cluster-ingress-operator has to work with.  I want to be clear that the LB management isn't new (i.e. it's already implemented) and isn't handled by cluster-ingress-operator (i.e. Istio handles it).  Would it be better to have a separate section to provide background on Istio's existing LB management logic, or would some kind of prefatory statement (e.g., "First, it is important to understand Istio's 'automated deployment' feature.")?

Reply:
> This section describes the implementation details, and I've kept it focused on the changes required in cluster-ingress-operator.

Yes, but this specific section, `New Controller to Manage DNS Records for Gateway Listeners`, is describing the DNS controller updates. My minor nit pick here, are these loadbalancer details feels a bit shoehorned into this section about DNS, and probably worth it's own section, just for better organization.

> Would it be better to have a separate section to provide background on Istio's existing LB management logic

Yea that's what I'm suggesting here. We can do it later (or never), it's just my thought on keeping things organized.

Reply:
I added more info about LB management to the "New Controller to Install and Configure OpenShift Service Mesh" section in https://github.com/openshift/enhancements/compare/732fe4ef4bfdf98263827650e5ca5127479bc8d1..5ccb87fd14dd84eebfc38506dc1596a75eeccdfb#diff-7973eca3c2f112ece8f8585f4f4c6567fc1c1b4fa95be7f88a27ce08820d89f7.